File: Leetcode/TwoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] outp = new int[2];
        for (int i = 0; i < nums.Length; i++) {
            for (int j = i + 1; j < nums.Length; j++) {
                if (nums[i] + nums[j] == target) {
                    outp[0] = i;
                    outp[1] = j;
                }
            }
        }
        return outp;
    }
}

File: Leetcode/Two_Sum.py
class Solution:
    def twoSum(self, nums, target):
        numDict = {}
        for index, num in enumerate(nums):
            if (target - num) not in numDict:
                numDict[num] = index
            else:
                return numDict[target - num], index
        return 0, 0


if __name__ == '__main__':
    nums = [2, 7, 11, 15]
    target = 9
    print(Solution().twoSum(nums, target))
